content/en/docs/ops/deployment/deployment-models/index.md

```diff
@@ -301,6 +273,24 @@ will likely be the best choice.
     caption="A service mesh with a primary and a remote cluster"
     >}}
 
+To support remote clusters in a multicluster mesh, the control plane in
+a primary cluster must be made accessible to them. For clusters spanning networks,
```

Comment:
Accessibility is only true for multi-network. For single network, it's about having a load balanced IP address for the control plane (so we don't have to use pod IP which is not stable).